index.html

```diff
@@ -245,6 +245,31 @@ <h5 class="skill">
         </ul>
       </section>
     </aside>
+
+    <!-- ---------- Contact me ---------- -->
+
+    <section class="myself">
+      <img src="image/contact form.png" alt="contact">
+      <aside class="touch">
+      <h2 class="last">Contact me</h2>
+      <p class="end">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+        I’d love to help with it!</p>
+      
+        <form class="form1">
```

Comment:
Nice job making your project match the given design😉. just as required for this milestone, kindly ensure your form makes use of the formspre service to retrieve data from clients😉.

<img width="1026" alt="image" src="https://github.com/VivekRaaZ/portfolio/assets/64643995/ca35a7e6-8d06-4fec-be76-9e8cf8c07f9f">